system/common.xml

```diff
@@ -350,14 +350,16 @@
 				</copy>
 			</then>
 		</if>
-		<condition property="isSVTTestRepo" value="true">
-			<isset property="env.isSVTTestRepo"/>
-		</condition>
-		<condition property="isSVTTestRepo" value="false">
-   		 	<not>
-				<isset property="env.isSVTTestRepo"/>
-			</not>
+		<property name="IS_SVT_TESTREPO" value="${env.IS_SVT_TESTREPO}"/>
+		<condition property="isSVTTestRepo" value="true" else="false">
+			<and>
+				<isset property="IS_SVT_TESTREPO"/>
+				<equals arg1="${IS_SVT_TESTREPO}" arg2="true"/>
+			</and>
 		</condition>
+		
+		<echo message="isSVTTestRepo is ${isSVTTestRepo}"/>
```

Comment:
Do we need both `IS_SVT_TESTREPO` and `isSVTTestRepo` variables?

Reply:
No we don't need both `IS_SVT_TESTREPO` and `isSVTTestRepo` variables.We can use only` IS_SVT_TESTREPO`